docs/WindowsBuild.md

```diff
@@ -1,408 +1,78 @@
 
-ninja -C S:\b\3
-```
-
-## Test Foundation
-
-```cmd
-ninja -C S:\b\3 test
-```
+> [!NOTE]
+> You only need the x64/x86 components when building for x64/x86 and vice versa. You need everything if you are cross-compiling.
 
-## Build swift-tools-core-support
+### Other dependencies
 
-```cmd
-cmake -B S:\b\5 ^
-  -D BUILD_SHARED_LIBS=YES ^
-  -D CMAKE_BUILD_TYPE=RelWithDebInfo ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_C_COMPILER=S:/b/1/bin/clang-cl.exe ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D SQLite3_INCLUDE_DIR=S:\Library\sqlite-3.28.0\usr\include ^
-  -D SQLite3_LIBRARY=S:\Library\sqlite-3.28.0\usr\lib\SQLite3.lib ^
-  -G Ninja ^
-  -S S:\swift-tools-support-core
+In a PowerShell shell, run the following commands to install the rest of the required tools.
 
-ninja -C S:\b\5
+```powershell
+winget install --id Git.Git --accept-package-agreements --accept-source-agreements
+winget install --id Python.Python.3.13 --accept-package-agreements --accept-source-agreements
+winget install --id Kitware.CMake --version 3.29.2 --accept-package-agreements --accept-source-agreements
+winget install --id Ninja-build.Ninja --version 1.13.1 --accept-package-agreements --accept-source-agreements
 ```
 
-## Build swift-llbuild
+> [!NOTE]
+> If Git and Python 3.7+ are already installed on your system, you don’t need to reinstall them here (i.e ignore the first 2 commands).
 
-```cmd
-cmake -B S:\b\6 ^
-  -D BUILD_SHARED_LIBS=YES ^
-  -D CMAKE_BUILD_TYPE=RelWithDebInfo ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_CXX_COMPILER=S:/b/1/bin/clang-cl.exe ^
-  -D CMAKE_CXX_FLAGS="-Xclang -fno-split-cold-code" ^
-  -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D LLBUILD_SUPPORT_BINDINGS=Swift ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D SQLite3_INCLUDE_DIR=S:\Library\sqlite-3.28.0\usr\include ^
-  -D SQLite3_LIBRARY=S:\Library\sqlite-3.28.0\usr\lib\sqlite3.lib ^
-  -G Ninja ^
-  -S S:\swift-llbuild
+## Windows configuration
 
-ninja -C S:\b\6
-```
+### Symbolic link creation permissions
 
-- Add llbuild to your path:
+Symbolic links are used in some of the swift repositories. You need to grant permission to your user to create symbolic links in order to properly clone the repositories:
 
-```cmd
-path S:\b\6\bin;%PATH%
-```
+Using `gpedit.msc` add your user to the list of trusted user in `Computer Configuration\Windows Settings\Security Settings\Local Policies\User Rights Assignment\Create Symbolic links`.
 
-## Build Yams
+In a PowerShell shell, run the following command to enable Git symbolic links and to disable automatic line endings conversion to support all the test suite:
```

Comment:
Again, these should work in either, I don't think that we should specify a shell here.